Description: Extend range of base dependency
Author: Joachim Breitner <nomeata@debian.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- haskell-template-0.2.0.5.orig/template.cabal
+++ haskell-template-0.2.0.5/template.cabal
@@ -1,5 +1,5 @@
 name:                template
-version:             0.2.0.5
+version:             0.2.0.5.1
 description:
   Simple string substitution library that supports \"$\"-based
   substitution.  Meant to be used when Text.Printf or string
@@ -19,7 +19,7 @@ library
   exposed-modules:  Data.Text.Template

   build-depends:
-    base >= 3.0 && < 4.5,
+    base >= 3.0 && < 4.6,
     mtl >= 1.1 && < 2.0.2,
     text >= 0.7.2 && < 0.12
